django/db/migrations/operations/special.py

```diff
@@ -110,12 +110,15 @@ def _run_sql(self, schema_editor, sql):
                         sql, params = sql
                     else:
                         raise ValueError("Expected a 2-tuple but got %d" % elements)
-                schema_editor.execute(sql, params=params)
-        else:
-            statements = schema_editor.connection.ops.prepare_sql_script(sql)
+                if sql != RunSQL.noop:
```

Comment:
Actually maybe not, I had `noop == function` in mind but that's for `RunPython`.